website/docs/d/network_routed.html.markdown

```diff
@@ -12,6 +12,10 @@ Provides a vCloud Director Org VDC routed Network data source. This can be used
 
 Supported in provider *v2.5+*
 
+-> **Note:** This data source supports only NSX-V backed Org VDC networks and is **deprecated**.
```

Comment:
What about using more eye-poking yellow box for these deprecations? Hopefully that would inspire more users to switch to the new data source.

Reply:
Changed. 
<img width="979" alt="image" src="https://user-images.githubusercontent.com/15804230/106649910-e4078880-659a-11eb-8ca2-e63e5e967eac.png">